### ISSUE TITLE
PR: add manual segmentations in the derivatives of `spine-generic-processed`

### ISSUE THREAD

**Issue by sandrinebedard** [neuropoly/data-management#111]
## Description
This PR adds 676 manual segmentations in the derivatives of `spine-generic-processed`on the branch  `sb/add-manual-seg`.

**Comment by sandrinebedard** [comment]
Hey @kousu, when you have a bit of time if you could take a look! Should be pretty straight forward 🙂  thanks!

**Comment by kousu** [comment]
* there were some `.DS_Store` accidental commits; I've gotten rid of them.
* the commit messages are slightly inaccurate: by my count, there's 674 additions, and 41 edits. I have squashed and change the commit message to match.
* `derivatives/labels/sub-amu04/anat/sub-amu04_acq-T1w_MTS_seg-manual.nii.gz` was missing its `.json` sidecar. I took a guess and copied the contents from `derivatives/labels/sub-amu04/anat/sub-amu04_T2w_seg-manual.json` into it.

Sorry for leaving you hanging for so long again -_-
